tests/integration/test_models/test_full_battery_models/test_lead_acid/test_composite.py

```diff
@@ -28,22 +28,20 @@ def test_optimisations(self):
         optimtest = tests.OptimisationsTest(model)
 
         original = optimtest.evaluate_model()
-        simplified = optimtest.evaluate_model(simplify=True)
         using_known_evals = optimtest.evaluate_model(use_known_evals=True)
-        simp_and_known = optimtest.evaluate_model(simplify=True, use_known_evals=True)
-        simp_and_python = optimtest.evaluate_model(simplify=True, to_python=True)
-        np.testing.assert_array_almost_equal(original, simplified)
+        simp_and_known = optimtest.evaluate_model(use_known_evals=True)
```

Comment:
this is now a duplicate of the above line (same in all other tests)

Reply:
I'm sorry I didn't realise that. I'll change that.